app/src/test/java/com/codesoom/assignment/infra/UserRepositoryTest.java

```diff
@@ -0,0 +1,49 @@
+package com.codesoom.assignment.infra;
+
+import static com.codesoom.assignment.domain.UserConstants.USER;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+@DisplayName("UserRepository 클래스")
+public class UserRepositoryTest {
+    @Autowired
+    UserRepository userRepository;
+
+    private User savedUser;
+
+    void subjectSave() {
+        savedUser = userRepository.save(USER);
+    }
```

Comment:
`save` 메소드의 리턴값이 테스트 대상이죠? 그렇다면 이렇게 해 줍시다.

```suggestion
    User subjectSave() {
        return userRepository.save(USER);
    }
```

Reply:
넵 수정하겠습니다.